src/vstart.sh

```diff
@@ -1114,7 +1114,7 @@ start_osd() {
 	if [ "$ceph_osd" == "crimson-osd" ]; then
         bottom_cpu=$(( osd * crimson_smp ))
         top_cpu=$(( bottom_cpu + crimson_smp - 1 ))
-	    # set a single CPU nodes for each osd
+	    # set exclusive CPU nodes for each osd
 	    extra_seastar_args="--cpuset $bottom_cpu-$top_cpu"
```

Comment:
Is there any difference to set "--cpuset" directly as part of the command, or to set through "$CEPH_BIN/ceph -c $conf_fn config set osd crimson_seastar_cpu_cores $bottom_cpu-$top_cpu"?

Reply:
We don't really want to use the seastar command line options directly.  Deployment tooling will need to pass information like this through the monitor config store, so we need to be able to set it from the ceph config system.  Allowing the sesatar command line options as well is likely to lead to confusion.  I think we should actually disallow them.

Reply:
Passing "--cpuset" will abort now, only crimson_seastar_cpu_cores is allowed.